src/project-1.js

```diff
@@ -2,139 +2,262 @@
+console.log(lessNine); 
 
 const greaterThanFifty = (num) => {
   // return true if num is greater than fifty
   // otherwise return false
-  // code here
+  // code here 
+  if (num > 50) {
+    return true;
+  } else {
+    return false;
+  }
 };
+const moreFifty = greaterThanFifty(45); 
+console.log(moreFifty);
 
 const add = (x, y) => {
   // add x and y together and return the value
-  // code here
+  // code here 
+  const add = x + y;
+  return add;
 };
+const sum = add(5, 6); 
+console.log(sum);
 
 const subtract = (x, y) => {
   // subtract y from x and return the value
-  // code here
+  // code here 
+  const minus = x - y; 
+  return minus; 
 };
+const min = subtract(3, 2); 
+console.log(min); 
 
 const divide = (x, y) => {
   // divide x by y and return the value
-  // code here
+  // code here 
+  const div = x / y; 
+  return div; 
 };
+const product = divide(35, 5); 
+console.log(product); 
 
 const multiply = (x, y) => {
   // multiply x by y and return the value
-  // code here
+  // code here 
+  const times = x * y; 
+  return times;
 };
+const answer = multiply(9, 9); 
+console.log(answer); 
 
 const getRemainder = (x, y) => {
   // return the remainder from dividing x by y
-  // code here
+  // code here 
+  const remain = x % y; 
+  return remain; 
 };
+const random = getRemainder(71, 5); 
+console.log(random); 
 
 const isEven = (num) => {
   // return true if num is even
   // otherwise return false
-  // code here
+  // code here 
+  if (num % 2 === 0) {
+    return true;
+  } else {
+    return false; 
+  }
 };
+const even = isEven(4); 
+console.log(even); 
 
 const isOdd = (num) => {
   // return true if num is odd
   // otherwise return false
-  // code here
-};
+  // code here 
+  if (num % 2 !== 0) {
+    return true;
+  } else {
+    return false; 
+  }
+}; 
+const odd = isOdd(5); 
+console.log(odd); 
 
 const square = (num) => {
   // square num and return the new value
   // code here
-};
+  const squ = num * num; 
+  return squ;
+}; 
+const shape = square(6); 
+console.log(shape); 
 
 const cube = (num) => {
   // cube num and return the new value
-  // code here
-};
+  // code here  
+  const cubed = num * num * num; 
+  return cubed;
+}; 
+const timesThree = cube(9); 
+console.log(timesThree); 
 
 const raiseToPower = (num, exponent) => {
   // raise num to whatever power is passed in as exponent
-  // code here
+  // code here 
+  const raise = Math.pow(num, exponent); 
+  return raise;
 };
+const power = raiseToPower(2, 3); 
+console.log(power); 
 
 const roundNumber = (num) => {
   // round num and return it
-  // code here
+  // code here 
+  const round = Math.round(num); 
+  return round; 
 };
+const round2 = roundNumber(0.5); 
+console.log(round2);
 
 const roundUp = (num) => {
   // round num up and return it
-  // code here
+  // code here  
+  const up = Math.ceil(num); 
+  return up
 };
+const round = roundUp(0.5); 
+console.log(round); 
 
 const addExclamationPoint = (str) => {
```

Comment:
defect: The requirements of the function are not met here. This function only returns whatever argument is passed to it.